File: Selenium/Activities/Activity_18.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.select import Select

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/selects")
    print("The page title is: ", driver.title)
    driver.maximize_window()
    drop_ele = driver.find_element(By.XPATH, "//select[@id='multi-select']")
    option = Select(drop_ele)
    option.select_by_visible_text("Javascript")
    option.select_by_index(4)
    option.select_by_index(5)
    option.select_by_index(6)
    option.select_by_value("node")
    option.deselect_by_index(5)
    time.sleep(2)
    driver.quit()

File: Selenium/Activities/Activity_18.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.Select;

public class Activity_18 {
    public static void main(String[] args) throws InterruptedException {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/selects");
        driver.manage().window().maximize();
        System.out.println("The pagetitle is: " + driver.getTitle());
        //Find element using locator
        WebElement option = driver.findElement(By.xpath("//select[@id='multi-select']"));
        Select select = new Select(option);
        select.selectByVisibleText("Javascript");
        Thread.sleep(2000);
        select.selectByIndex(4);
        select.selectByIndex(5);
        select.selectByIndex(6);
        select.selectByValue("node");
        Thread.sleep(2000);
        select.deselectByIndex(5);
        Thread.sleep(2000);
        driver.quit();
    }
}
